_sub/compute/k8s-traefik-flux/main.tf

```diff
@@ -0,0 +1,18 @@
+resource "github_repository_file" "traefik_kustomization" {
+  repository = var.repo_name
+  branch     = local.repo_branch
+  file       = "${local.base_repo_path}/kustomization.yaml"
+  content    = jsonencode(local.kustomization)
+}
+
+resource "github_repository_file" "traefik_patch" {
+  repository = var.repo_name
+  branch     = local.repo_branch
+  file       = "${local.base_repo_path}/patch.yaml"
+  content    = jsonencode(local.patch)
+}
+
+resource "kubectl_manifest" "traefik_fallback" {
+    count       = var.fallback ? 1 : 0
```

Comment:
Could this variable be named fallback_enabled or something to clarify it's purpose better.